src/PriceIndicator/PriceIndicator.tsx

```diff
@@ -60,11 +67,12 @@ export const PriceIndicator: React.FC<Props> = ({ total, addition, isActive = fa
             {children}
           </HoverContainer>
         )}
-        <TotalPrice isActive={isActive} style={{ zIndex: 2 }}>
+
+        <TotalPrice isActive={isActive} isPriceHover={isPriceHover}>
```

Comment:
Wanna walk me through why we need to differentiate if its the pricing indicator?

Reply:
If you have a look at TotalPrice in the .styles file, it uses that to set the border radius'. I can show you on screen if you like?

Reply:
Could you change `isPriceHover` to `isPriceHovered`? I think when it is isPriceHover is slightly confusing